package.json

```diff
@@ -47,7 +47,7 @@
     "compile:code": "swc src -d dist && swc src -d dist/cjs -C module.type=commonjs && echo '{\"type\":\"commonjs\"}' > dist/cjs/package.json",
     "compile:types": "tsc --project tsconfig.build.json",
     "test": "tsc --project . --noEmit",
-    "e2e": "playwright test",
+    "e2e": "cd e2e/01 && playwright test && cd ../02 && playwright test",
```

Comment:
It doesn't seem ideal, but let's go with it for now. If you find a better solution, it would be nice.